src/Grpc/JsonTranscoding/src/Microsoft.AspNetCore.Grpc.JsonTranscoding/Internal/Binding/JsonTranscodingProviderServiceBinder.cs

```diff
@@ -244,6 +244,10 @@ private static CallHandlerDescriptorInfo CreateDescriptorInfo(string body, strin
         FieldDescriptor? responseBodyDescriptor = null;
         if (!string.IsNullOrEmpty(responseBody))
         {
+            if (responseBody.Contains('.', StringComparison.Ordinal))
```

Comment:
Are dot separators the only way users might think to pass nested references. Not sure if we can expect indexed access like `foo["bar"]`?

Reply:
dot separators are the only way.